docs/development/local_setup.md

```diff
@@ -124,35 +144,24 @@ While WSL1, plain docker for windows and various Linux distributions and local K
 
 The Gardener repository and all the above-mentioned tools (git, golang, kubectl, ...) should be installed in your WSL2 distro, according to the distribution-specific Linux installation instructions.
 
-## [Optional] Installing gcloud SDK
+# Start Gardener locally
 
-In case you have to create a new release or a new hotfix of the Gardener you have to push the resulting Docker image into a Docker registry. Currently, we are using the Google Container Registry (this could change in the future). Please follow the official [installation instructions from Google](https://cloud.google.com/sdk/downloads).
-
-## Local Gardener setup
-
-This setup is only meant to be used for developing purposes, which means that only the control plane of the Gardener cluster is running on your machine.
-
-### Get the sources
+## Get the sources
 
 Clone the repository from GitHub into your `$GOPATH`.
 
 ```bash
 mkdir -p $GOPATH/src/github.com/gardener
 cd $GOPATH/src/github.com/gardener
-git clone git@github.com:gardener/gardener.git
+git clone https://github.com/gardener/gardener.git
```

Comment:
What is wrong with the git protocol?

Reply:
Oops, I didn't want to PR this change. I was thinking if https makes it easier for first-time contributors, but wanted to remove this in the end. Thanks for catching it!